doc/MAPGEN.md

```diff
@@ -994,6 +994,29 @@ an `update_mapgen`, as normal mapgen can just specify the terrain directly.
 
 - "transform": (required, string) the id of the `ter_furn_transform` to run.
 
+### Spawn nested chunks based on overmap neighbors with "place_nested"
+
+Place nested allows for limited conditional spawning of chunks based on the `"id"` of its overmap neighbors.  This is useful for creating smoother transitions between biome types, or to dynamically create walls at the edges of a mutable structures.
+
+| Field              | Description
+| ---                | ---
+| chunks/else_chunks | (required, string) the nested_mapgen_id of the chunk that will be conditionally placed. Chunks are placed if the specified neighbor matches, and "else_chunks" otherwise.  
+| x and y            | (required, int) the cardinal position in which the chunk will be placed. 
+| neighbors          | (optional, string ) Any of the neighboring overmaps that should be evaluated before placing the chunk. Despite the plural field name, only a single neighbor direction can be evaluated per chunk.  The direction itself can check for any amount of overmap `"id"` substrings.
+|
+
+Any of the 26 adjacent overmaps can be evaluated in this manner.  The direct cardinal neighbors ( `"north", "east", "south", "west"` ) the inter cardinal neighbors ( `"north_east", "north_west", "south_east", "south_west"` ), the direct vertical neighbors ( `"above", "below"` ) and combinations of the vertical and cardinal neighbors ( `"above_north", "above_north_east"`... etc).
```

Comment:
Can all 26 actually be used for this? @jbytheway?

Reply:
Updated, they cant.